yeastphenome/apps/conditions/models.py

```diff
@@ -271,14 +249,10 @@ def papers_all(self):
         return ps
 
     def papers_link_list(self):
```

Comment:
This is one of the papers_link_list not used, should we remove or add to some admin view?